packages/ssr-web/config/environment.js

```diff
@@ -11,20 +11,28 @@ const universalLinkHostnamesByTarget = {
   production: MERCHANT_PAYMENT_UNIVERSAL_LINK_HOSTNAME,
 };
 
-const CARD_SPACE_LOCAL_DEV_HOSTNAME_SUFFIX = '.card.space.localhost';
+const CARD_SPACE_LOCAL_DEV_HOSTNAME_SUFFIX = '.card.xyz.localhost';
 const CARD_SPACE_STAGING_HOSTNAME_SUFFIX = '.pouty.pizza';
 const CARD_SPACE_HOSTNAME_SUFFIX = '.card.xyz';
 
 const cardSpaceHostnameSuffixByTarget = {
   staging: CARD_SPACE_STAGING_HOSTNAME_SUFFIX,
   production: CARD_SPACE_HOSTNAME_SUFFIX,
 };
+
+function convertHostnameSuffixToRegex(suffix) {
+  return `/^.+\\${suffix}$/`;
```

Comment:
this doesn't escape the full stops, which should still work but could technically match a set of other domains.

Reply:
true 😯 fixed in b0be65a, thank you!